package.json

```diff
@@ -109,6 +109,9 @@
     "get-changed-validators": "dist/beacon-light-client/plonky2/validators_commitment_mapper_tree/get_changed_validators.js"
   },
   "dependenciesMeta": {
+    "@lodestar/types@1.17.0": {
+      "unplugged": true
```

Comment:
Just curious: why does it need to be unplugged?

Reply:
Somewhere in the code, we infer types from this lib. But I hit the following error at some point:
`This is likely not portable. A type annotation is necessary.`. So my solution was to unplug it and tell TS where to look for the types. See commit `config(relay): Add @lodestar/types in paths.`

ref:https://yarnpkg.com/cli/unplug#details

Reply:
Oh, got it, thanks.